config/routes.rb

```diff
@@ -54,6 +54,7 @@ def matches?(request)
       get "/:host_type/:owner/:name/dependencies", to: "repositories#dependencies", constraints: { name: /[^\/]+/ }
       get "/:host_type/:owner/:name/shields_dependencies", to: "repositories#shields_dependencies", constraints: { name: /[^\/]+/ }
       get "/:host_type/:owner/:name/projects", to: "repositories#projects", constraints: { name: /[^\/]+/ }
+      get "/:host_type/:owner/:name/sync", to: "repositories#sync", constraints: { name: /[^\/]+/ }
```

Comment:
I don't know why all the API endpoints are `get`, but I went with convention. Since they're behind api tokens and don't need cookie auth, they're safe from CSRF.